themes/gtk/theme.lua

```diff
@@ -44,25 +30,9 @@ local function is_dark(color_value)
     local is_dark_bg = (bg_numeric_value < 383)
     return is_dark_bg
 end
-local function mix(color1, color2, ratio)
-    ratio = ratio or 0.5
-    local result = "#"
-    local channels1 = color1:gmatch(hex_color_match)
-    local channels2 = color2:gmatch(hex_color_match)
-    for _ = 1,3 do
-        local bg_numeric_value = math.ceil(
-          tonumber("0x"..channels1())*ratio +
-          tonumber("0x"..channels2())*(1-ratio)
-        )
-        if bg_numeric_value < 0 then bg_numeric_value = 0 end
-        if bg_numeric_value > 255 then bg_numeric_value = 255 end
-        result = result .. string.format("%02x", bg_numeric_value)
-    end
-    return result
-end
 local function reduce_contrast(color, ratio)
-    ratio = ratio or 50
-    return darker(color, is_dark(color) and -ratio or ratio)
+    ratio = (ratio or 50) / 100
+    return gcolor.darken(color, is_dark(color) and ratio or 0)
```

Comment:
it was before `-ratio or ratio` but after your change `ratio or 0`

Reply:
Ah, I was thinking if it was dark then don't do anything, I guess if it's dark then the original `darker` made it lighter?